2025-01-31 21:43:31,861:DEBUG:certbot._internal.main:certbot version: 3.1.0
2025-01-31 21:43:31,862:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2025-01-31 21:43:31,862:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/var/www/certbot', '--email', 'karimmaroous@gmail.com', '--agree-tos', '--no-eff-email', '--staging', '-d', 'opsforall.net']
2025-01-31 21:43:31,863:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-01-31 21:43:31,892:DEBUG:certbot._internal.log:Root logging level set at 30
2025-01-31 21:43:31,896:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2025-01-31 21:43:31,896:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A separate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x725db324af00>
Prep: True
2025-01-31 21:43:31,896:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x725db324af00> and installer None
2025-01-31 21:43:31,896:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2025-01-31 21:43:32,060:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-staging-v02.api.letsencrypt.org/acme/acct/182784314', new_authzr_uri=None, terms_of_service=None), 5c0ea7419c19db18f607881e60bb62df, Meta(creation_dt=datetime.datetime(2025, 1, 31, 21, 18, 24, tzinfo=datetime.timezone.utc), creation_host='f5a16ff59e49', register_to_eff=None))>
2025-01-31 21:43:32,064:DEBUG:acme.client:Sending GET request to https://acme-staging-v02.api.letsencrypt.org/directory.
2025-01-31 21:43:32,070:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-staging-v02.api.letsencrypt.org:443
2025-01-31 21:43:32,261:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 974
2025-01-31 21:43:32,262:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:43:32 GMT
Content-Type: application/json
Content-Length: 974
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "Rz4vqMGFXak": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-staging-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "profiles": {
      "classic": "The same profile you're accustomed to",
      "tlsserver": "https://letsencrypt.org/2025/01/09/acme-profiles/"
    },
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org/docs/staging-environment/"
  },
  "newAccount": "https://acme-staging-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-staging-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-staging-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-staging-v02.api.letsencrypt.org/acme/revoke-cert"
}
2025-01-31 21:43:32,263:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for opsforall.net
2025-01-31 21:43:32,266:DEBUG:acme.client:Requesting fresh nonce
2025-01-31 21:43:32,268:DEBUG:acme.client:Sending HEAD request to https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce.
2025-01-31 21:43:32,329:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-01-31 21:43:32,330:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:43:32 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xatTN6bVwoyhsZ_U0qA1NVzlnqewwOFu74PJTXJOoEWHHs
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-01-31 21:43:32,330:DEBUG:acme.client:Storing nonce: xMYr_xatTN6bVwoyhsZ_U0qA1NVzlnqewwOFu74PJTXJOoEWHHs
2025-01-31 21:43:32,330:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "opsforall.net"\n    }\n  ]\n}'
2025-01-31 21:43:32,334:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXRUTjZiVndveWhzWl9VMHFBMU5WemxucWV3d09GdTc0UEpUWEpPb0VXSEhzIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "2OTzxUlcSzynDFE_FYfhAHDUjl9ad4nsoxA-sEZ034ieVwWk9PhzTKAc_2yPzFm2m41-I66gpx6jHPv4x4uynwmCLAFC1XmuoXn1vQgO7ku8pLIvwR9FY6M5uutpjX4wqPs5ur_3ny0jTGO-Xp4YM3alwar3aO5tRSGUztG1eGc2Udmd9VnTpI1YE70q-jbft9mhKa8TLgtCn2eqbs76lmv_LJLoPIkP5H4Hjrk7Dwj2_eNiO174M2qeUgZb-mguo2Fmepr1FCNDEeppF5R2B-7gsLNht8dii3441iu0Y4_NDkvFofnIMNtGzWrTF3CtIkIgd2NGTOU1piKHQcX1sA",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogIm9wc2ZvcmFsbC5uZXQiCiAgICB9CiAgXQp9"
}
2025-01-31 21:43:32,428:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 359
2025-01-31 21:43:32,430:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Fri, 31 Jan 2025 21:43:32 GMT
Content-Type: application/json
Content-Length: 359
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/order/182784314/22333517074
Replay-Nonce: xMYr_xat19M7txoB5lG6ZjX5afnpEHMR7f3-cQdptGue7HTzuBc
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-02-07T21:43:32Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "opsforall.net"
    }
  ],
  "authorizations": [
    "https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902276234"
  ],
  "finalize": "https://acme-staging-v02.api.letsencrypt.org/acme/finalize/182784314/22333517074"
}
2025-01-31 21:43:32,431:DEBUG:acme.client:Storing nonce: xMYr_xat19M7txoB5lG6ZjX5afnpEHMR7f3-cQdptGue7HTzuBc
2025-01-31 21:43:32,432:DEBUG:acme.client:JWS payload:
b''
2025-01-31 21:43:32,435:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902276234:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXQxOU03dHhvQjVsRzZaalg1YWZucEVITVI3ZjMtY1FkcHRHdWU3SFR6dUJjIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjI3NjIzNCJ9",
  "signature": "YLumIOyjSuaJFSo3sb1s8ewziH_Fm2Eha9RbWzom5XN-iPL1vtKhuA4ZuGoKjEagYE86VxnNL6hQL9rpB21ldQofiiHituadnU1RdywhiunacNJCa-PkTxbfpX-XaTlVIUnWuDY7b51Sr10VPJ-Kgg7elnlCGbngo2ZGX63PaXm8Z8jTysu018mF05UB6xD-uSTZjqNLLE9YCBheC7YDD5OjhmpLMTEg3OwdTFvQmMh0jbQIzujeVlor-iVKodt23KxH0CLgC-ADzOa9ngKmWYoZuAXFyJjWJMP3Ix18KxgDMiRABlLcstyyC77R5kejFHIJjteXBwNYw9RxfbMLKA",
  "payload": ""
}
2025-01-31 21:43:32,503:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902276234 HTTP/1.1" 200 839
2025-01-31 21:43:32,507:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:43:32 GMT
Content-Type: application/json
Content-Length: 839
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xatKVcYgfwKYp5kPp3_UNFnvMOoiQrWQMazXpa48uELTGg
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "pending",
  "expires": "2025-02-07T21:43:32Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902276234/i3qKhQ",
      "status": "pending",
      "token": "dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU"
    },
    {
      "type": "dns-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902276234/OGGp3w",
      "status": "pending",
      "token": "dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902276234/dh3oRw",
      "status": "pending",
      "token": "dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU"
    }
  ]
}
2025-01-31 21:43:32,507:DEBUG:acme.client:Storing nonce: xMYr_xatKVcYgfwKYp5kPp3_UNFnvMOoiQrWQMazXpa48uELTGg
2025-01-31 21:43:32,507:INFO:certbot._internal.auth_handler:Performing the following challenges:
2025-01-31 21:43:32,508:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 21:43:32,508:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2025-01-31 21:43:32,508:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2025-01-31 21:43:32,509:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU
2025-01-31 21:43:32,510:DEBUG:acme.client:JWS payload:
b'{}'
2025-01-31 21:43:32,514:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902276234/i3qKhQ:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXRLVmNZZ2Z3S1lwNWtQcDNfVU5GbnZNT29pUXJXUU1helhwYTQ4dUVMVEdnIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzE4Mjc4NDMxNC8xNTkwMjI3NjIzNC9pM3FLaFEifQ",
  "signature": "M-nky1Q551Obn969LxDIwMZgqI730EvmYC-32tccROrc-MIe8vi6idFHsJUGllqukuhBGGHYD4pKnL65s5A19DwzF4z8DrbFqrD2CTls6O6JGwxvtSskBIMQBIYS79FAlFSA_CWCgPrcSYzDi-FDV5sQQqa4GAVb0AkQJECk203mfpfGeKjbVQeUNBdRUOLN1n0en9SutSazZ9le7I99X9Su5aXm-C5X2b5qzSqs5iWsNTtrCLTHFyZq1yOpxKSulQ11kIMk1F13rgQZYXJvunH3KKdE8BtJGI4G26usG7M1GrKut-_xOtS1K0CIhHJXVnc_4D6gJk4b_Q2uIWATwA",
  "payload": "e30"
}
2025-01-31 21:43:32,580:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/chall/182784314/15902276234/i3qKhQ HTTP/1.1" 200 201
2025-01-31 21:43:32,581:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:43:32 GMT
Content-Type: application/json
Content-Length: 201
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902276234>;rel="up"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902276234/i3qKhQ
Replay-Nonce: N9Ep_yiTLn8Sar0-Zv-XcszwigPLJSSaYXoClVDl_B0c1Po2u4E
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902276234/i3qKhQ",
  "status": "pending",
  "token": "dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU"
}
2025-01-31 21:43:32,583:DEBUG:acme.client:Storing nonce: N9Ep_yiTLn8Sar0-Zv-XcszwigPLJSSaYXoClVDl_B0c1Po2u4E
2025-01-31 21:43:32,583:INFO:certbot._internal.auth_handler:Waiting for verification...
2025-01-31 21:43:33,584:DEBUG:acme.client:JWS payload:
b''
2025-01-31 21:43:33,585:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902276234:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVRMbjhTYXIwLVp2LVhjc3p3aWdQTEpTU2FZWG9DbFZEbF9CMGMxUG8ydTRFIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjI3NjIzNCJ9",
  "signature": "3wGgQQ3ncSDAgwsSoaWjFlBks0PmWYr_wXwSxt8E5UF4KWnNa_ov5lcWN_YNm9vuVrhForlXna-dkRp4O0B7-VzjcrnD2cQhu5GcuEEvaGVKq-nVs9HmYXn6y6M3moLFfy69Kox9xgkEMkPeti4NywOMSqgbsOOgxTeN9a1-u_f7FbBnh5gguFdGgibvV80g6BZRlS3SOTQ4dyuA9SM21GLmPPs03WKTrnHH5QnC-SyLBGtJVCW5Tw2oGfTSoqQU6_TMvfE3im6TV1UMzlmOb3Qf4Jg3slshBHZlqK_nOj0p4PTq28tpsR0N5MzbH23MAW8zkiL-xJJRbRMQDkzh4g",
  "payload": ""
}
2025-01-31 21:43:33,654:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902276234 HTTP/1.1" 200 1264
2025-01-31 21:43:33,654:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 21:43:33 GMT
Content-Type: application/json
Content-Length: 1264
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xatJE2i2QFOf8EvZ91lcYX8o7XGiH34S3mRulTlPaWrN3g
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "invalid",
  "expires": "2025-02-07T21:43:32Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902276234/i3qKhQ",
      "status": "invalid",
      "validated": "2025-01-31T21:43:32Z",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "54.226.228.6: Invalid response from http://opsforall.net/.well-known/acme-challenge/dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU: \"\u003c!DOCTYPE html\u003e\u003chtml lang=\\\"en\\\"\u003e\u003chead\u003e\\n  \u003cmeta charset=\\\"utf-8\\\"\u003e\\n  \u003ctitle\u003eAngularFrontend\u003c/title\u003e\\n  \u003cbase href=\\\"/\\\"\u003e\\n  \u003cmeta name=\\\"\"",
        "status": 403
      },
      "token": "dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU",
      "validationRecord": [
        {
          "url": "http://opsforall.net/.well-known/acme-challenge/dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU",
          "hostname": "opsforall.net",
          "port": "80",
          "addressesResolved": [
            "54.226.228.6"
          ],
          "addressUsed": "54.226.228.6"
        }
      ]
    }
  ]
}
2025-01-31 21:43:33,655:DEBUG:acme.client:Storing nonce: xMYr_xatJE2i2QFOf8EvZ91lcYX8o7XGiH34S3mRulTlPaWrN3g
2025-01-31 21:43:33,655:INFO:certbot._internal.auth_handler:Challenge failed for domain opsforall.net
2025-01-31 21:43:33,655:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 21:43:33,655:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: opsforall.net
  Type:   unauthorized
  Detail: 54.226.228.6: Invalid response from http://opsforall.net/.well-known/acme-challenge/dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU: "<!DOCTYPE html><html lang=\"en\"><head>\n  <meta charset=\"utf-8\">\n  <title>AngularFrontend</title>\n  <base href=\"/\">\n  <meta name=\""

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2025-01-31 21:43:33,656:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2025-01-31 21:43:33,656:DEBUG:certbot._internal.error_handler:Calling registered functions
2025-01-31 21:43:33,656:INFO:certbot._internal.auth_handler:Cleaning up challenges
2025-01-31 21:43:33,656:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/dKcRskZaYv54HzadYO-MgP3akqWZlIJS_jVv-WTwmMU
2025-01-31 21:43:33,656:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2025-01-31 21:43:33,656:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1873, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1579, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 142, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 518, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 429, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 497, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2025-01-31 21:43:33,659:ERROR:certbot._internal.log:Some challenges have failed.
